### PR TITLE
PERFT-100:  refactor browser metrics

### PR DESCRIPTION
**Pull request by Kryvetskyi**

### CODE REVIEW

File: app/util/confluence/browser_metrics.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+metrics = {
```
Comment (SergeyMoroz0703) [inline app/util/confluence/browser_metrics.py:2]: Please rename it to `browser_metrics`, to be precise.

File: app/selenium_ui/conftest.py [hunk 2]
```diff
@@ -246,19 +246,22 @@ def get_requests_by_url(requests, url_path):
     return filtered_requests
 
 
-def get_wait_browser_metrics(webdriver):
+def get_wait_browser_metrics(webdriver, expected_metrics):
     attempts = 20
```
Comment (SergeyMoroz0703) [inline app/selenium_ui/conftest.py:250]: Please reduce the attempts to 10, to have a total possible sleep time 5 sec. And need to retest it with 5 secs and 10 attempts.

File: app/selenium_ui/conftest.py [hunk 3]
```diff
@@ -246,19 +246,22 @@ def get_requests_by_url(requests, url_path):
...
-    for i in range(0, attempts):
+    for i in range(attempts):
         requests = get_performance_logs(webdriver)
         requests_bulk = get_requests_by_url(requests, 'bulk')
-        for request_id, request in requests_bulk.items():
-            if 'browser.metrics.navigation' in str(request):
-                return requests_bulk
+        data.update(requests_bulk)
+
+        if all([metric in str(data) for metric in expected_metrics]):
```
Comment (SergeyMoroz0703) [inline app/selenium_ui/conftest.py:259]: Maybe it is worth collecting the `keys` from the data and checking if the metric is in the keys? Need to test and discuss.

File: app/selenium_ui/conftest.py [hunk 4]
```diff
@@ -246,19 +246,22 @@ def get_requests_by_url(requests, url_path):
...
+            return data
+
         print(f'Waiting for browser metrics, attempt {i}, sleep {sleep_time}')
         time.sleep(sleep_time)
+
```
Comment (SergeyMoroz0703) [inline app/selenium_ui/conftest.py:264]: If metrics are not found need to print a WARNING message, like `WARNING: expected_metrics not found for the some_action`

File: app/selenium_ui/conftest.py (hunk 2 above)
Comment (Kryvetskyi) [reply app/selenium_ui/conftest.py]: Doesn't work properly with 10 attempts. Tested with 15 attempts, looks good to me.

**Review by SergeyMoroz0703 — APPROVED** [review]

**Pull request merged**